packages/manager/apps/telecom/src/app/error-page/error-page.module.js

```diff
@@ -0,0 +1,12 @@
+import managerErrorPage from '@ovh-ux/manager-error-page';
+
+import routing from './error-page.routing';
+
+const moduleName = 'ovhManagerTelecomErrorPage';
```

Comment:
Why are you using a `Page` suffix only for Telecom et Web application?

Because for both Cloud and Dedicated, the module was named as:  `ovhManagerCloudError` and `ovhManagerDedicatedError`.

Reply:
I like it better without the `Page` too :)